editor/plugins/theme_editor_plugin.cpp

```diff
@@ -2350,7 +2416,13 @@ void ThemeTypeEditor::_update_type_list() {
 
 	theme_type_list->clear();
 
-	if (theme_types.size() > 0) {
+	if (theme_types.is_empty()) {
+		theme_type_list->set_disabled(true);
+		theme_type_list->add_item(TTRC("None"));
```

Comment:
```suggestion
		theme_type_list->add_item(TTRC("None"));
		theme_type_list->set_item_auto_translate_mode(AUTO_TRANSLATE_MODE_ALWAYS);
```
The list has disabled translation.